docs/Doxyfile

```diff
@@ -280,7 +280,7 @@ OPTIMIZE_OUTPUT_VHDL   = NO
 # parses. With this tag you can assign which parser to use for a given
 # extension. Doxygen has a built-in mapping, but you can override or extend it
 # using this tag. The format is ext=language, where ext is a file extension, and
-# language is one of the parsers supported by doxygen: IDL, Java, Javascript,
+# language is one of the parsers supported by doxygen: IDL, Java, JavaScript,
```

Comment:
Some projects have asked me to drop changes to `Doxyfile` as it tends to be just copied from elsewhere. I'm happy to do that if requested (or similarly for other files) -- just let me know.

Reply:
I think it's okay; eventually I'll switch to using the CMake 3.1x Doxygen tools, but until then, I don't think the Doxyfile will get regenerated.